test/11_array.jl

```diff
@@ -32,6 +32,8 @@ for isordered in (false, true)
         @test convert(CategoricalVector{String, DefaultRefType}, x) == x
         @test convert(CategoricalVector{String, UInt8}, x) == x
 
+        @test get(x[2]) == a[2]
```

Comment:
Actually, I'd rather put these tests into `value.jl`. The array tests are already complex enough.